econml/orf/_ortho_forest.py

```diff
@@ -1300,8 +1300,12 @@ def effect_inference(self, X=None, *, T0=0, T1=1):
             a dataframe summary of these inference results.
         """
         eff, scales = self._effect_inference_helper(X, T0, T1)
+        # d_t=1 here since we measure the effect across all Ts
+        # A dummy treatment names is passed in
```

Comment:
Consider rewording, since it is not actually passed in

```suggestion
        # Omit treatment name argument to fall back to the default
```